src/components/thumbnail-card/ThumbnailCard.js

```diff
@@ -8,6 +8,7 @@ import ThumbnailCardThumbnail from './ThumbnailCardThumbnail';
 import './ThumbnailCard.scss';
 
 type Props = {
+    actionItem?: React.Node,
```

Comment:
Could this type be more specific, e.g., `React.Element`?

Reply:
Good call out!